window.go

```diff
@@ -34,6 +35,7 @@ const (
 	EventNameWindowCmdUnmaximize               = "window.cmd.unmaximize"
 	EventNameWindowCmdWebContentsCloseDevTools = "window.cmd.web.contents.close.dev.tools"
 	EventNameWindowCmdWebContentsOpenDevTools  = "window.cmd.web.contents.open.dev.tools"
+	EventNameWindowCmdWebContentsExecutedJavaScript = "window.event.web.contents.executed.javascript"
```

Comment:
This constant should be `EventNameWindowCmdWebContentsExecuteJavaScript = "window.cmd.web.contents.execute.javascript"`